src/js/juju-cards-v1.0.0.js

```diff
@@ -60,6 +60,12 @@ let jujuCards = () => {
     }
 
     let addLink = `${demoDomain}/?deploy-target=${getImageID(id)}`;
+    let deployTitle = 'Deploy with Juju';
+
+    if (typeof(card.dataset.dd) !== 'undefined') {
+      addLink = `${demoDomain}/?dd=${getImageID(id)}`;
+      deployTitle = 'Direct Deploy with JAAS';
```

Comment:
Shouldn't both cases simply be "Deploy with JAAS"? Does that mean that the only thing the option does is change the link? 

My objection with adding complexity is that we (JAAS team) should make a choice either we want external sites to DD or add to canvas. 

If we go for the dual option approach we will need to document this in the readme of this project and also probably add the option to the cards page on jujucharms.com.